src/content/app/genome-browser/services/browser-storage-service.ts

```diff
@@ -99,6 +104,18 @@ export class BrowserStorageService {
     this.storageService.update(StorageKeys.TRACK_PANELS, trackPanels);
   }
 
+  public getBrowserSidebarModals(): {
+    [genomeId: string]: Partial<BrowserSidebarModalState>;
+  } {
+    return this.storageService.get(StorageKeys.SIDEBAR_MODAL) || {};
+  }
+
+  public updateBrowserSidebarModals(browserSidebarModals: {
+    [genomeId: string]: Partial<BrowserSidebarModalStateForGenome> | undefined;
+  }): void {
+    this.storageService.update(StorageKeys.SIDEBAR_MODAL, browserSidebarModals);
+  }
```

Comment:
I forgot — did Andrea really want us to store which modals were open for which species? So that if you reload the browser the modal is still open?

Reply:
So now that Andrea said she doesn't want the open/closed state of the modal to survive browser refresh, what are we storing here, and should we be storing anything at all?

Reply:
We are using this to store the previously viewed objects for Bookmarks. So probably should go into the new bookmarks storage service.